site/layouts/partials/scripts.html

```diff
@@ -9,6 +9,10 @@
 <script src="https://cdn.jsdelivr.net/npm/@stackblitz/sdk@1/bundles/sdk.umd.js"></script>
 {{- end }}
 
+{{ if eq .Page.Layout "single" -}}
```

Comment:
Isn't this duplicated in another layout file?

If so, we should deduplicate common code because it's a significant amount of it. Ideally, the inline JS should go into external JS files, but it won't be easy if we need Hugo to process them.

So, at least deduplicating common StackBlitz layout code this would be an improvement.